package/MDAnalysis/analysis/pca.py

```diff
@@ -0,0 +1,270 @@
+        If a subset of components is not chosen then all components are stored
+        and the sum of explained variances is equal to 1.0.
+    start: int
+        The index of the first frame to be used for the creation of the
+        covariance matrix.
+    stop: int
+        The index to stop before in the creation of the covariance matrix.
+    step: int
+        The amount of frames stepped between in the creation of the covariance
+        matrix.
+    Methods
+    -------
+    fit(traj=None, select='All', start=None, stop=None, step=None)
+        Find the principal components of selected atoms in a subset of the
+        trajectory.
+    transform(traj=None, n_components=None)
+        Take the original trajectory and project it onto the principal
+        components.
+    inverse_tranform(pca_space)
+        Take a pca_space and map it back onto the trajectory used to create it.
+    """
+
+    def __init__(self, atomgroup, select='all', align=False, mean=None,
+                 n_components=None, **kwargs):
+        """
+        Parameters
+        ----------
+        atomgroup: MDAnalysis atomgroup
+            AtomGroup to be used for PCA.
+        select: string, optional
+            A valid selection statement for choosing a subset of atoms from
+            the atomgroup.
+        align: boolean, optional
+            If True, the trajectory will be aligned to a reference
+            structure.
+        mean: MDAnalysis atomgroup, option
+            An optional reference structure to be used as the mean of the
+            covariance matrix.
+        n_components : int, optional
+            The number of principal components to be saved, default saves
+            all principal components, Default: None
+        start : int, optional
+            First frame of trajectory to use for generation
+            of covariance matrix, Default: None
+        stop : int, optional
+            Last frame of trajectory to use for generation
+            of covariance matrix, Default: None
+        step : int, optional
+            Step between frames of trajectory to use for generation
+            of covariance matrix, Default: None
+        """
+        super(PCA, self).__init__(atomgroup.universe.trajectory,
+                                  **kwargs)
+        self._u = atomgroup.universe
+        if self._quiet:
+            logging.disable(logging.WARN)
+        # for transform function
+        self.align = align
+        # access 0th index
+        self._u.trajectory[0]
+        # reference will be 0th index
+        self._reference = self._u.select_atoms(select)
+        self._atoms = self._u.select_atoms(select)
+        self.n_components = n_components
+        self._n_atoms = self._atoms.n_atoms
+        self._calculated = False
+        if mean is None:
+            logger.warn('In order to demean to generate the covariance matrix\n'
+                        + 'the frames have to be iterated over twice. To avoid\n'
```

Comment:
no need for the plus.

Please use a text like this. It importantly directly states how a mean structure can be provided.

```
A mean structure will be calculated to calculate the covariance matrix.
This means the trajectory will be iterated twice. You can also provide a mean
structure with the "mean" argument
```